### DISCUSSION TITLE
tkinter.TclError: image "pyimage1" doesn't exist

### DISCUSSION THREAD

**Discussion by axelprsy** [TomSchimansky/CustomTkinter#1091] [Q&A]
Hello, when trying to put an image in my application, it returns me: `_tkinter.TclError: image "pyimage1" doesn't exist`
Do you have a solution to the problem?

Code : 

```python
image_path = os.path.join(os.path.dirname(os.path.realpath(__file__)))
self.product_image = customtkinter.CTkImage(dark_image=Image.open(os.path.join(image_path, "product-image.png")), size=(500, 150), light_image=Image.open(os.path.join(image_path, "product-image.png")))

self.product_image_label = customtkinter.CTkLabel(self.frame_1, text="", image=self.product_image)
self.product_image_label.pack(padx=10, pady=10)
```

Error : 

```
self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: image "pyimage1" doesn't exist
```

**Comment by tomivanek** [comment]
I have a same problem, any possible solution?

**Reply by avalon60** [reply]
Have you tried printing the result of the os.path.join?

**Comment by pasabanov** [comment]
Found a solution in https://github.com/TomSchimansky/CustomTkinter/discussions/1479#discussioncomment-5655698.

Instead of calling `ctk.CTk()` you should create new windows with `CTkTopLevel()`.
